### ISSUE TITLE
review J.2 2nd vide, create / list

### ISSUE THREAD

**Issue by rodislav** [rodislav/Mister_JavaBean_Lessons#33]
hello, please take a look at this video: https://www.youtube.com/watch?v=RT6HHvqxm3M&index=4&list=PLPkoWZmDIKwAMCtfB_PI1ffcXX1_Rzxai

**Comment by andreicoval** [comment]
- I suggest you to describe what each class/interface is suppose to do in your application and why you decided to spread functionalities this way? I assume that a junior developer is not able at first glance to perceive the logic behind it. This will significantly reduce the time necessary to understand the code;

Below, I enumerated few new topics, which wasn't covered until this stage. I think a short description on them will be enough for better understanding:
- import! - if we instantiate a class from different package, than we must import this class to our code: import packageName (for ex.: import junior.contactList.view.ConsoleView);
- method as a parameter for another method! - we can pass as a parameter to a method another method only if the Data Type of expected parameter is the same as the return type of the method passed as parameter;
- "toString()" override - what happened if not overrided?
- "press any key to continue" message - changed "return scanner.nextInt()"(which generated an error) by a try/catch block with "Integer.parseInt(scanner.next())";

In conclusion, this is a very good material for this stage, as it has a mix of known and new topics, the vidoe length is not an issue from my point of view and it's quite challenging for a junior developer to understand all the tricks from this code.

**Comment by andreicoval** [comment]
In addition, it's not clear why ContactsService class and ContactsRepository interface were created. It seems they have the same functionality. Isn't enough to have only the interface with 2 implementations, without ContactsService class?
